### ISSUE TITLE
[Bug]: ContextualMenu Accessibility Issue aria-required-children

### ISSUE THREAD

**Issue by v-yiyun** [microsoft/fluentui#28343]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Package version: @fluentui/react: 8.110.5
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu

## Repro Steps
1. Navigate to the documentation for the ContextualMenu component: https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu
2. Find the "ContextualMenu with header" example
3. Open the contextual menu
4. Run the Accessibility Insights for Web fast pass.

## Actual Behavior
Implementations of the FluentUI ContextualMenu are currently failing the Accessibility Insights for Web fast pass. See screenshots and attached logs.
![Screenshot (93)](https://github.com/microsoft/fluentui/assets/137975389/617f63d6-040c-46fc-aece-0838f23cfe6c)
![Screenshot (91)](https://github.com/microsoft/fluentui/assets/137975389/f886709c-5fab-408c-86a6-e8b963745a3f)

## Expected Behavior
The ContextualMenu component should pass the Accessibility Insights for Web fast pass without any accessibility issues.

### Logs

```shell
Title: WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (#id__304-menu)
Tags: Accessibility, WCAG 1.3.1, aria-required-children

Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://accessibilityinsights.io/info-examples/web/aria-required-children)

Target application: Fluent UI - Controls - React - ContextualMenu - https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu#fluent-ui-v9

Element path: #id__304-menu

Snippet: <div id="id__304-menu" class="ms-ContextualMenu-container container-247" tabindex="0" aria-labelledby="id__304" role="menu">

How to fix: 
Fix any of the following:
  Element has children which are not allowed (see related nodes)
  Element has no aria-busy="true" attribute

Environment: Microsoft Edge version 114.0.1823.58

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by behowell** [comment]
This is a false positive by Accessibility Insights. The item under Related Paths that it's complaining about is an icon that has `aria-hidden="true"`, which is not in the accessibility tree and should not be triggering this error.

The issue in axe-core is logged here: https://github.com/dequelabs/axe-core/issues/3850

It is fixed in axe-core v4.7.1, but Accessibility Insights is still using v4.6.3.
